tests/test_helper.tcl

```diff
@@ -474,8 +474,7 @@ proc read_from_test_client fd {
     } elseif {$status eq {err}} {
         set err "\[[colorstr red $status]\]: $data"
         puts $err
-        set test_name [lindex [split $data "\n"] 0]
-        lappend ::failed_tests $test_name
```

Comment:
we can print the whole err actually, it have more info

Reply:
Oh so you are referring to the print at the end of the test? we do have a print when the test is reporting the error during the run right?

Reply:
yes, the line 476 print the log, it happen during the run.

and here, the `lappend ::failed_tests`, we will print it at the end of the test.